### DISCUSSION TITLE
Different aired_episodes count in Show Summary and watched progress

### DISCUSSION THREAD

**Discussion by Kashish-Sharma** [trakt/trakt-api#334] [Q&A] [answered]
There's a difference in count of `aired_episodes`,

For `https://api.trakt.tv/shows/attack-on-titan/progress/watched?extended=full`
count is `81`

For `https://api.trakt.tv/shows/attack-on-titan?extended=full`
count is `80`

Is this expected ?

**Comment by tysonkerridge** [comment, marked as answer]
This might help: https://github.com/trakt/api-help/issues/17#issuecomment-457280529

Basically it's due to caching
